tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>"At this stage, you know how to manipulate SPAM errors and laser noises. In this <strong>multi-qubit</strong> case, we will sum up all these effects."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>